### PR TITLE
Dataset page

### PR DESCRIPTION
**Pull request by Samad198**
# Changes
- Edited the dataset admin page to show immutable values if the user is not an admin
- Added a link to the dataset admin page in the scanreport list page

# Migrations
# Screenshots

If the user is not an admin, they will see this
image
Otherwise, they will see
image

# Checks

**Important:** please complete these **before** merging.
- [ ] Run migrations, if any.
- [ ] Update `changelog.md`, including migration instructions if any.
- [ ] Run unit tests.

### CODE REVIEW

**Review by spco — APPROVED** [review]
LGTM. I wonder whether we should change the URL as `datasets/admin` is a bit misleading when one isn't admin, but that is a separate concern from this PR.

**Comment by spco** [comment]
Once this is merged, please double-check that everything is working on ccom-dev.

**Comment by spco** [comment]
And just make sure the overlap with #429 works - slightly concerned they may both implement the functionality to test for admin status to allow edits to some fields.

**Review by dcl10 — COMMENTED** [review]
LGTM. I'll try and merge this stuff with mine after this is merged.

**Pull request merged**
